src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -634,7 +640,8 @@ export default class FilingHistoryList extends Vue {
 
       // add properties for staff filings
       if (this.isTypeStaff(filing)) {
-        item.documents = [] // no documents
+        // court orders may have documents so leave property as null (they load on toggle)
+        if (!this.isTypeCourtOrder(filing)) item.documents = [] // no documents
```

Comment:
It's my preference to use braces for if statements, even if they're one liners. Not sure if we have a standard for this, though.

Reply:
I can go either way. Lead the way if you feel strongly about this.

What about code like this?
`if (!val) return // safety check`

Reply:
I'm OK with that, too. Any other reviewers have an opinion?

Do you want this line of code updated or is it OK as-is? (Just looking to resolve this conversation.)

Reply:
LGTM. Chris, please resolve this conversation if you are OK with it.

Reply:
I don't want to dictate style. 😎 I'm okay to leave it as is.

Reply:
We are the team. You have a say! :)